DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/platform_designer_standard/README.md

```diff
@@ -0,0 +1,401 @@
+
+   4. Choose **Empty Project** when prompted to select a project type.
+
+   5. Add the source file `add.sv` and `jtag.sdc` to the design when the wizard prompts you. These may be copied from `starting_files`.
+
+      ![](assets/add-files.png)
+
+      `add.sv` is the top-level entity of this design. It instantiates an instance of the Platform Designer system that contains your simple SYCL HLS IP.
+      
+      `jtag.sdc` contains timing constraints for the JTAG IP.
+
+3. Copy the IP you generated in Step 1 to the Intel Quartus® Prime project. 
+
+   Linux:
+
+   ```bash
+   $> cd .. # navigate to build root if not there already
+   $> cp -r build/add.report.prj/ add_quartus/
+   ```
+
+   Windows:
+
+   ```bash
+   > cd .. # navigate to build root if not there already
+   > ROBOCOPY build\add.report.prj\ add_quartus\add.report.prj\ /S /NFL /NDL
+   ```
+
+4. Create the Platform Designer system.
+
+   1. Open Platform Designer from the Intel® Quartus® Prime GUI. A a blank system will be created for you. ![](assets/new-platform-designer-system.png)
+
+   2. Add the following IP to your system:
+
+      * Basic Functions > Bridges and Adaptors > Memory Mapped > **JTAG to Avalon Master Bridge Intel® FPGA IP**
+
+      * oneAPI > **add_report_di**
+
+      ![](assets/add-ip-platform-designer.png)
+
+      > **Note**: If you cannot see the oneAPI IP component, refresh Platform Designer by clicking `File` > `Refresh System`
+      > ![](assets/refresh-system.png)
+
+   3. Connect the modules as shown:
+
+      ![](assets/module-connections.png)
+
+      Don't forget to export the `irq_add` and `exception_add` signals. The provided top-level RTL file (`add.sv`) uses the generated IP. Following these naming conventions allows the IP to connect to this handwritten RTL.
+
+   4. Save the system by clicking `File` > `Save` and name it `add_kernel_wrapper.qsys`.
+
+   5. Make sure there are no errors in the 'System Messages' panel.
+
+   6. Generate the system so that it can be included in the Intel® Quartus® Prime project by clicking `Generate HDL...`
+
+      ![](assets/generate-hdl.png)
+
+   7. Close Platform Designer. 
+   
+5. In the Intel® Quartus® Prime window, run Analysis and Elaboration by clicking 'Start Analysis and Elaboration'.
+
+   ![](assets/start-analysis.png)
+
+6. Select pins for the `i_clk` and `reset_button_n` inputs and `fpga_led` output. The JTAG to Avalon® Master Bridge Intel FPGA IP handles the connection between your design and the JTAG pins on your board automatically.
+
+   1. Open the pin planner using `Assignments` > `Pin Planner` in the main Intel® Quartus® Prime GUI. In the bottom pane in Pin Planner GUI, you will find a list of signals to be assigned a pin. Consult the data sheet for your board to choose an appropriate clock input. Double-click the *Location* cell and select a proper clock input. In this project, the `PIN_AF14` was chosen because it supplies a 50MHz clock signal.
```

Comment:
```suggestion
   1. Open the pin planner using `Assignments` > `Pin Planner` in the main Intel® Quartus® Prime GUI. In the bottom pane of the Pin Planner GUI, you will find a list of signals to assign pins to. Consult the data sheet for your board to choose an appropriate clock input. Double-click the *Location* cell and select a proper clock input. In this project, the `PIN_AF14` was chosen because it supplies a 50MHz clock signal.
```